hedera-platform/build.gradle.kts

```diff
@@ -23,18 +23,22 @@ val besuVersion = "23.1.2"
 val bouncycastleVersion = "1.70"
 val daggerVersion = "2.42"
 val eclipseCollectionsVersion = "10.4.0"
-val grpcVersion = "1.45.1"
-val helidonVersion = "3.0.2"
-val jacksonVersion = "2.13.3"
+val grpcVersion = "1.54.1"
+val helidonVersion = "3.2.1"
+val jacksonVersion = "2.13.5"
 val log4jVersion = "2.17.1"
 val mockitoVersion = "4.6.1"
-val nettyVersion = "4.1.77.Final"
-val protobufVersion = "3.19.4"
+val nettyVersion = "4.1.87.Final"
+val protobufVersion = "3.21.7"
 val swirldsVersion = "0.39.0-alpha.3"
 val systemStubsVersion = "2.0.2"
 val testContainersVersion = "1.17.2"
 val tuweniVersion = "2.2.0"
 
+dependencies {
+    api(enforcedPlatform("io.netty:netty-bom:$nettyVersion"))
```

Comment:
Note: If we want to publish this at some point, we should rethink this.

It would be better to use `platform` in place of `enforcedPlatform`. `enforce` is very "strong". It was added in Gradle to mimic Maven's "BOM import" behavior. It's ok if you are using it as the "final consumer". Which we are as long as we do not publish this for others. 

Since `enforcedPlatform` has an effect in the current dependency setup, I would leave it like this. But we might want to revisit at some point.

Reply:
I'm definitely fine with relaxing this constraint, I originslly took the bulldozer approach to get it working. I'm good with being a bit more surgical about it if it works to resolve the versioning mismatches.

Reply:
Confirmed that reducing the constraint to a `platform()` instead of `enforcedPlatform()` results in mixed version netty dependencies.

Reply:
There may be another way to refactor this to not need the stronger enforcement but at this point I would consider it outside the scope of this PR.